### ISSUE TITLE
Default Ctrl+alt+arrows hotkey - already an existing hotkey in most software

### ISSUE THREAD

**Issue by itskando** [rxhanson/RectanglePro-Community#22]
Ctrl+alt+arrows is a weird default, since that combo is fairly universal to any text box in any software, allowing for word jump.
Shift+ctrl+alt+arrows is similar - word jump highlighting/selection.

Recommendation:

Make Ctrl+Cmd+arrows default (unless you can think of an existing universal hotkey for that too).
Ctrl+Cmd + mouse movement is already a standard for window throw, so this checks out.

(Cmd could become quick throw if you wanted Cmd to be consistently associated with the app, but that's outside the scope here.)

**Comment by itskando** [comment]
Added notes:

Ctrl+Cmd+Return: System hotkey for search in Music.
Ctrl+Cmd+Space: Emoji menu.

No perfect catch-all for hotkey modifiers.

Keyboards universally need an additional modifier key XD

**Comment by rxhanson** [comment]
Since Rectangle (and therefore Rectangle Pro) came after a number of other solutions out there, and of those solutions, it seemed that either ctrl+option or cmd+option were the defaults. It felt weird to abandon those entirely, especially coming along after so many users had already gotten accustomed to the prior apps as it's a lot easier to pull market share away if users don't have to relearn a new set of defaults. When I initially wrote Rectangle, I actually just kind of felt like most people would just record all their own shortcuts but it's obvious that most people don't like to stray from the defaults or find it painful. Perhaps adding the "master" control as mentioned in your other issue would alleviate some of that.

There's not really an easy way to see where all the conflicts are going to occur either. Honestly, it might make sense to build out a set of alternate defaults that's just built for the hyper key. I have thought about just building Hyperkey into Rectangle Pro (or Multitouch), but it feels like feature creep to do so. And I have a really large backlog of more applicable features for Rectangle Pro and Multitouch anyway.

**Comment by itskando** [comment]
Yeah, after a bit more research (seen above and in other issues I posted):

The Fn key, which I thought was just an additional modifier button, standardized and recognized by macOS, appears to be more unique to apple laptop embedded keyboards and thus cannot be relied on as a default since anyone with a mac mini or third-party keyboard will be left out..

Ctrl+Alt+Arrows is taken for text across all OS.
Ctrl+Cmd+[Enter/Space] is taken for macOS.
So there's no sweeping modifier combo that could be spammed across all of your existing defaults.

.

Given that there is no modifier which could be used consistently for all present hotkey non-modifiers, the present combinations continue to make sense, although Ctrl+Alt+Arrows remains a bit iffy since it's already in heavy use, but I see above and hear that seamlessness between competitors is a real draw.

.

I'll have to see how well HyperKey works on external keyboards. Thanks for the tip.

My external keyboard's fn key cannot produce the fn input that the standard macbook pro embedded laptop produces.
Since it doesn't provide any input .. I can't even turn it into something else.

Maybe I can turn the caps lock key into the fn and salvage the external .. but the dead fn key kills me, especially since it's a dead key and thus has no use - but was placed between the ctrl and alt keys rather than on the corner edge, making combining ctrl with alt or cmd fairly awkward. (not your problem.)
